source/compatibility.tex

```diff
@@ -64,6 +64,31 @@
                                 // changed according to the global C locale
 \end{codeblock}
 
+\rSec2[diff.cpp23.containers]{\ref{containers}: containers library}
+
+\diffref{span.overview}
```

Comment:
Is it typical to only specify the class synopsis
instead of the parent subclause that also encompasses the detailed specifications?

Reply:
They did the same thing for cpp20, so I guess so?